src/niswitch/examples/niswitch_connect_channels.py

```diff
@@ -18,7 +18,7 @@ def example(resource_name, channel1, channel2, topology, simulate):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Performs a connection with NI-SWITCH Channels.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Switch.')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI-SWITCH.')
```

Comment:
"NI-SWITCH" is the name of a software product.
An "NI switch" is a switch made by NI.

Looking at all these, I realize we should not capitalize the instrument type unless it's an initialism (DMM, SMU).

Can you fix please?

Reply:
Updated